overview/glossary.md

```diff
@@ -11,35 +11,82 @@ sidebarType: 0
 - TOC
 {:toc}
 
-## Account ID
+## Ad Formats
 
-`Prebid Server` - Prebid Server host companies may give out account IDs to publishers for authorization, configuration,
-and reporting purposes. This account ID is provided on s2sConfig when utilizing Prebid Server from Prebid.js and in the init function when utilizing Prebid SDK.
+This section contains terms around ad formats and their respective configuration.
 
-## AdSlot
+### MediaType or Media Format
 
-The 'hole' in the web page or mobile app where an ad may appear. This is very similar to `AdUnit`, but is
-the technical implementation defining which ad sizes and formats may appear within it.
+`banner` - also called 'display', these ads are generally simple rectangular images.
 
-In the "Google Publisher Tag" (GPT), the "defineSlot()" function call maps an HTML div (the technical
-implementation) to the AdUnit, (the sellable item).
+`native` - the components of this mediatype like text, links, and images are all separated so they
+can be formatted to smoothly fit into the content.
 
-## AdUnit
+`video` - see Player-Initiated Video and Page-Initiated Video
 
-`Ad servers such as Google Ad Manager` - the AdUnit is the thing that business people care about. It's an
-inventory concept targeted by [line items](#line-items).
+### Multiformat AdUnit
 
-[Prebid Adunit] - Prebid adds another layer to the ad server AdUnit: which header bidders should participate in the auction and their parameters.
+An AdUnit that can display creatives of multiple media formats: banner, native, and/or video.
+
+### Page-Initated Video
+
+Also called 'outstream', these are AdUnits that work in Prebid.js just like banners, but are displayed with
+a special 'renderer' to play a video ad instead of a static image.
+
+### Player-Initiated Video
+
+Once called 'instream', these are AdUnits that live within a video player.
```

Comment:
Should we link to https://github.com/InteractiveAdvertisingBureau/AdCOM/blob/develop/AdCOM%20v1.0%20FINAL.md#list_plcmtsubtypesvideo and mention the new `plcmt`  video subtypes as well here?

I'm not to familiar with it, but @karimMourra and @fowler446 are. WDYT guys?

Reply:
I'd say plcmt can be described elsewhere; `instream` doesn't entirely map to a value in plcmt since there are other considerations.

Reply:
Not sure if "lives within a video player" is the right terminology since video players can render non linear ads which are "in-video banner ads" essentially. 
I would suggest
```suggestion
### Page-Initated Video

Also called 'outstream', these are AdUnits that work in Prebid.js just like banners, but are displayed with
a special 'renderer' to play a video ad instead of a static image. Said renderer only renders the ad and does not render content.

### Player-Initiated Video

Once called 'instream', these are AdUnits that are rendered in a video player before, during or after content.
```